frontend/coprs_frontend/coprs/templates/_helpers.html

```diff
@@ -157,6 +157,12 @@
     <span class="pficon pficon-ok"></span> {{ state }}
   {% elif state == "failed" %}
     <span class="pficon pficon-error-circle-o"></span> {{ state }}
+    {% if failed_build_log_url and config.LOG_DETECTIVE_BUTTON %}
```

Comment:
The config option is ON in prod, and we want to keep this disabled for some time... I think we need a new option.

Reply:
Maybe not, it will be quite some time before we'll do a new Copr release anyway.